test_project/adam.py

```diff
@@ -101,3 +101,13 @@ def single_iteration():
         result = True
 
     return result
+
+
+def raise_exception():
```

Comment:
No, you're absolutely right. I wasn't paying enough attention! Sorry about that.